mpls/draft-ietf-detnet-mpls.xml

```diff
@@ -709,31 +708,27 @@ DFx = DetNet member flow x over a TE LSP
        provisioned via configuration, e.g., via the controller plane
        described by
        <xref target="I-D.ietf-detnet-data-plane-framework"/>.
-       Implementations MAY required that PEF and POF be used in combination.
+       Implementations MAY require that PEF and POF be used in combination.
        There is no requirement related to the order of execution of the Packet
        Elimination and Ordering Functions in an implementation.
      </t>
      <t>
-       After a DetNet service is identified for a received DetNet MPLS
-       packet, as described above, an implementation MUST check if POF
-       is configured for that DetNet service.  When configured, the
-       implementation MUST track the sequence number contained in
-       received d-CWs and MUST ensure that packets are processed in the
-       order indicated in the received d-CW sequence number field, which
-       may not be in the order the packets are received. As defined in
+	   After a DetNet service is identified for a received DetNet MPLS packet,
+	   as described above, if POF is configured for that DetNet service, packets

```

Comment:
Is it "if a POF" ?  if a Packet Ordering Function vs if Packet Ordering Function

Reply:
My favorite is "if POF", I will keep it.